SPECS/prometheus-adapter/prometheus-adapter.spec

```diff
@@ -1,18 +1,22 @@
 Summary:        Kubernetes Custom, Resource, and External Metric APIs implemented to work with Prometheus.
 Name:           prometheus-adapter
 Version:        0.10.0
-Release:        6%{?dist}
+Release:        7%{?dist}
 License:        Apache-2.0
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://github.com/kubernetes-sigs/prometheus-adapter
 Source0:        https://github.com/kubernetes-sigs/%{name}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
 BuildRequires:  golang
-BuildRequires:  prometheus
 
 %description
 Implementation of Prometheus via Kubernetes Custom, Resource, and External Metric API.
 
+%package docs
+Summary: prometheus-adapter docs
```

Comment:
I would create a dependency on the matching ver-rel of the main package. That way, docs will stay updated with the main package.

Reply:
In this case, would the other comment about a license not apply?

Reply:
Since you added the dependency, we don't need the licenses.

Reply:
Makes sense. Thanks!